### PR TITLE
adding missing Azure facts modules

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Several modules for DTL

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
azure_rm_devtestlabschedule_facts
azure_rm_devtestlabpolicy_facts
azure_rm_devtestlavenvironment_facts
azure_rm_devtestlabcustomimage_facts

##### ADDITIONAL INFORMATION

Currently DTL Tests are disabled, so the modules were tested manually.

### CODE REVIEW

File: lib/ansible/modules/cloud/azure/azure_rm_devtestlabcustomimage_facts.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+try:
+    from msrestazure.azure_exceptions import CloudError
+    from azure.mgmt.devtestlabs import DevTestLabsClient
+    from msrest.serialization import Model
+except ImportError:
+    # This is handled in azure_rm_common
+    pass
+
+
+class AzureRMCustomImageFacts(AzureRMModuleBase):
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_devtestlabcustomimage_facts.py]: DevTestLabCustomImage?

File: lib/ansible/modules/cloud/azure/azure_rm_devtestlabenvironment_facts.py [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+try:
+    from msrestazure.azure_exceptions import CloudError
+    from azure.mgmt.devtestlabs import DevTestLabsClient
+    from msrest.serialization import Model
+except ImportError:
+    # This is handled in azure_rm_common
+    pass
+
+
+class AzureRMEnvironmentFacts(AzureRMModuleBase):
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_devtestlabenvironment_facts.py]: class name with devtestlab?

File: lib/ansible/modules/cloud/azure/azure_rm_devtestlabcustomimage_facts.py [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+            type: str
+            sample: "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxxx/resourcegroups/myResourceGroup/providers/microsoft.compute/snapshots/myImage"
+        source_vm_id:
+            description:
+                - Source VM id.
+            returned: always
+            type: str
+            sample: "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxxx//resourcegroups/myResourceGroup/providers/microsoft.devtestlab/labs/myLab/v
+                     irtualmachines/myLabVm"
+        tags:
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_devtestlabcustomimage_facts.py:106]: it has tags, can facts be filtered by tags?

**Review by yungezz — COMMENTED** [review]
general good, some comments

**Comment by Fred-sun** [comment]
@zikalino Need you update according by the comments! Thanks!

**Comment by zikalino** [comment]
@yungezz added "Dtl" to class names, also added facts.
performed all manual tests

**Comment by yungezz** [comment]
> @yungezz added "Dtl" to class names, also added facts.
> performed all manual tests

seems you didn't push

**Review by yungezz — APPROVED** [review]

**Pull request merged**